test_package/conanfile.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import re
+import subprocess
+
+from conan.tools.cmake import CMake
+from conans import ConanFile
+
+
+class VehicleAppSdkTest(ConanFile):
+    name = "vehicle_app_sdk_test"
+    license = "Apache-2.0"
+    url = "https://github.com/eclipse-velocitas/vehicle-app-cpp-sdk"
+    
+    settings = "os", "compiler", "build_type", "arch"
+    
+    generators = "CMakeToolchain", "CMakeDeps"
+
+    def requirements(self):
+        self.requires(self.tested_reference_str)
+    
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        pass
```

Comment:
shouldn't we at least try to run the app once? to really test if the import is working?

Reply:
Added a test for this.